content/03.results.md

```diff
@@ -2,41 +2,48 @@
+
+### TODO Section
+
+We defined cancer groups, broad histologies, etc. as follows for the rest of the paper.
 
 ### Molecular Subtyping of OpenPBTA CNS Tumors
 
-Over the past two decades, together with the World Health Organization (WHO), experts in neurooncology have iteratively redefined the classifications of central nervous system (CNS) tumors [@doi:10.1093/jnen/61.3.215; @doi:10.1007/s00401-007-0243-4]. 
-More recently, in 2016 and 2021 [@doi:10.1007/s00401-016-1545-1; @doi:10.1093/neuonc/noab106], molecular subtypes have been integrated into these entities. 
-The Children’s Brain Tumor Tissue Consortium (CBTTC), currently the Children’s Brain Tumor Network (CBTN), opened its protocol for brain tumor and matched normal sample collection in 2011 and as such, the majority of the samples within the OpenPBTA dataset lack molecular subtype annotation. 
-In the absence of methylation arrays commonly used for tumor classification, we coupled key genomic features of tumor entities described by the WHO in 2016 and low-grade glioma (LGG) subtypes described by Ryall and colleagues [@doi:10.1016/j.ccell.2020.03.011] with clinician and pathologist review to subtype 64% (1,281/2,007) of tumor biospecimens with high confidence (**Table S1**).
-Importantly, this collaborative molecular subtyping process allowed us to identify data entry errors (e.g., an ETMR entered as a medulloblastoma) and mis-identified specimens (e.g., Ewing sarcoma sample labeled as a craniopharyngioma), update diagnoses using modern terms (e.g., primitive neuro-ectodermal tumor [PNET] diagnoses), and discover rarer tumor entities within the OpenPBTA (e.g., H3-mutant ependymoma, meningioma with _YAP1::FAM118B_ fusion).
-**Table 1** lists the subtypes we defined within the OpenPBTA, comprising of LGATs (N = 569), high-grade gliomas (N = 314), embryonal tumors (N = 229), ependymomas (N = 65), tumors of sellar region (N = 51), mesenchymal non-meningothelial tumors (N = 21), glialneuronal tumors (N = 20), and chordomas (N = 12). 
+Over the past two decades, experts in neurooncology have worked with the World Health Organization (WHO) to iteratively redefine the classifications of central nervous system (CNS) tumors [@doi:10.1093/jnen/61.3.215; @doi:10.1007/s00401-007-0243-4]. 
+More recently, in 2016 and 2021 [@doi:10.1007/s00401-016-1545-1; @doi:10.1093/neuonc/noab106], molecular subtypes have been integrated into these classifications. 
+In 2011, the Children’s Brain Tumor Tissue Consortium (CBTTC), now known as the Children’s Brain Tumor Network (CBTN), opened its protocol for brain tumor and matched normal sample collection.
```

Comment:
"opened its protocol" is not a phrase I am familiar with, so I've assumed it's a clinical phrase I don't know and did not modify it. Thoughts?